src/styles/images/index.md.njk

```diff
@@ -8,53 +8,144 @@ layout: layout-pane.njk
 
-## Alternative text
+## Illustrations
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+Use illustrations to show a simplified image of something complex. For example:
+- to demonstrate a physical interaction, like scanning a passport
+- to help users find something on a document, like a reference number
+
+### Creating illustrations
+
+Use as few visual elements as possible to communicate your message.
+
+Start by using the [GOV.UK colour palette](/styles/colour/#colour-palette). If you need more colours, try to use tints of the colour palette.
+
+Use flat blocks of colour. Avoid using shadows or gradients, except when needed to show a distinctive feature of an object or to communicate layering or depth.
+
+If creating multiple illustrations for a service, use consistent colours, line widths and border radiuses across them all.
+
+Avoid depicting people in illustrations. If you need to show people, think carefully about how you represent the people that use your service — do not only show one kind of person. 
+For example, only depicting people of a particular age, gender or ethnicity can exclude people, reinforce negative stereotypes and cause harm.
+
+Some examples of illustrations from services:
+
+## Icons
+
+Avoid using icons in most cases. People can understand icons to mean different things, so they can be confusing.
+
+Icons can be more useful in case working systems, where users are familiar with the interface and return to it frequently. In this context, they can help users to scan pages more quickly. In most cases it’s still helpful to include a text label alongside any icons.
+
+Do not use a single icon to represent more than one thing. For example, the search icon (magnifying glass) should only be used for search functionality, and not also for screen magnification.
+
+## Avoid images that contain text
+
+Use written content instead of an image that contains text, wherever possible.
+
+Any text inside an image will be difficult to read, especially for users that need to read text using:
+- screen readers
+- copy and paste
+- highlight
+- resize
+- recolour
+
+However, text can sometimes be an inseparable part of an image, especially in photography. Examples of this might be a photo with a signpost in it, or a company's name on the side of a building.
+
+If you choose to use an image that contains text, you should also include written content nearby that conveys the same meaning. This means the information is available to everyone
+
+Only use alt text as a last resort, when you need to use an image with text that cannot be removed, replaced or supplemented by written content. Be sure to follow the guidance on this page about writing good alt text.
```

Comment:
As mentioned before, not writing the text of images of text into the body copy would be a WCAG fail. I cannot imagine a case where an image of text cannot be "removed, replaced or supplemented". This should not just be discouraged. Although this is specifically about images of text and not all images with text on them.

Reply:
Same here, I wrote this before I got to the section 'Avoid images of text'. I wonder if it's generally clear to people what the difference is between the two sections.